manifests/n/NickeManarin/ScreenToGif/2.37.0/NickeManarin.ScreenToGif.installer.yaml

```
@@ -0,0 +1,35 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: NickeManarin.ScreenToGif
+PackageVersion: 2.37.0
```

```suggestion
PackageVersion: 2.37
```
This should match control panel